security/transport/transaction.py

```diff
@@ -80,3 +82,40 @@ def is_active_logged_requests(using=None):
     :return: True if block of code is surrounded with atomic_log operator
     """
     return hasattr(get_connection(using), 'output_logged_requests') and get_connection(using).output_logged_requests
+
+
+
+def get_all_request_related_objects():
+    """
+    Return all related selected with log_request_related_objects decorator.
+    :return: list of request related objects
+    """
+    connection = get_connection()
+    related_objects_list = getattr(connection, 'related_objects_list', [])
+    return list(itertools.chain(*related_objects_list))
```

Comment:
I am not getting this statement - why don't you just return `related_objects_list`? Making a generator to evaluate it as a list immediately does not make sense to me....

Reply:
you can use context processor inside context processor

```
with log_request_related_objects(user):
    requests.get # log user to the logged request
    with log_request_related_objects(user2):
        request.get # log user,user in the request

    request.get # log user in the logged request


```

the function must return all related objects according t

Reply:
viz test test_output_logged_request_should_be_related_with_object_selected_in_decorator

Reply:
the list() will convert it immediately, though? but the test works so :monkah: